ftw/linkchecker/profiles/default/metadata.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0"?>
 <metadata>
     <dependencies>
+        <dependency>profile-plone.app.dexterity:default</dependency>
```

Comment:
This might be unwanted - not sure if you really want to install Dexterity everywhere that the link-checker gets installed. But that should be reviewed by someone else, im not involved enough in these details of the project.

Reply:
👍